db/migrate/20170505035229_migrate_mailing_list_daily_updates_users_to_daily_summary.rb

```diff
@@ -0,0 +1,11 @@
+class MigrateMailingListDailyUpdatesUsersToDailySummary < ActiveRecord::Migration
+  def change
+    change_column_default :user_options, :mailing_list_mode_frequency, 1
+
+    UserOption.exec_sql(<<~SQL)
+    UPDATE user_options
+    SET digest_after_minutes = 1440, email_digests = 't', mailing_list_mode = 'f'
+    WHERE mailing_list_mode_frequency = 0 AND mailing_list_mode
+    SQL
```

Comment:
@SamSaffron  Can you review the changes in this migration? I would like an extra pair of :eyes:  on this. Thank yoU!